src/Symfony/Component/HttpClient/Messenger/PingWebhook.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpClient\Messenger;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+final class PingWebhook implements \Stringable
```

Comment:
Should we suffix this class with `Message` (`PingWebhookMessage`), like suggested in https://github.com/symfony/symfony/pull/49813#pullrequestreview-1357880462?

Reply:
`find src/ -name '*Message.php'|grep Tests -v` says we should, isn't it?

Reply:
Yep, made this change.